flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
why limiting the arch to x86_64 only?

Reply:
I am not sure tbh, can I just delete the flathub.json?